templates/asp-app.json

```diff
@@ -174,7 +174,7 @@
       "resources": [
         {
           "name": "[concat('tm', parameters('additional_host_name'))]",
-          "condition": "[not(equals(parameters('is_frontend'), '0'))]",
+          "condition": "[not(contains(createArray('', 'null', 'false', 'localhost', 'debugparam'), parameters('additional_host_name')))]",
```

Comment:
Invalid values for `additional_host_name` are based on current usage of this parameter within HMCTS repos.